client/src/modules/search-results/index.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import { Card, Grid, Typography } from '@material-ui/core';
+import Layout from '../home/shared/components/Layout';
+
+const SearchResults: React.FC = () => {
+  // TODO: Replace with `useEventsQuery()`!
+  const data = {
+    searchResults: [],
+    searchTerm: 'London',
+  };
+
+  return (
+    <Layout>
+      <h1>
+        Search results for <q>{data?.searchTerm}</q>
+      </h1>
+      <Grid container spacing={2}>
+        <Grid item xs={10}>
+          {data?.searchResults.length === 0 && (
+            <Typography gutterButtom variant="p" component="p">
```

Comment:
@Ryuno-Ki Two warnings on build: 

1. `gutterButtom` (misspelled). We can use `paragraph` instead...?
2. `variant` doesn't accept `p` as value as per Material API. `component=p` alone might do the job. 

@timmyichen would you mind reviewing this...? 

I just did the build to see how it looks.

Reply:
Note to self: https://material-ui.com/components/typography/#changing-the-semantic-element

Reply:
@Ryuno-Ki what's the point of assigning the variant to be a paragraph if the component is itself a paragraph...? 

According to my understanding of that doc, `variant` prop applies styling of the passed value. `component` prop makes the node of that kind. In our case variant and component are same. 

For situations where we need a `<h2>` and the styles of `<h1>` in `<h2>`, variant might be of help. But here simply passing `p` to component will do the job.

Reply:
List of variants: https://material-ui.com/customization/typography/#variants